test/lib/code_corps/validators/slug_validator_test.exs

```diff
@@ -88,6 +88,21 @@ defmodule CodeCorps.Validators.SlugValidatorTest do
     refute changeset.valid?
   end
 
+  test "reserves all api routes" do
+    CodeCorpsWeb.Router.__routes__
+    |> Enum.map(&Map.get(&1, :path))
+    |> Enum.map(&String.split(&1, "/"))
+    |> List.flatten
+    |> Enum.reject(fn fragment -> fragment == "" end)
+    |> Enum.reject(fn fragment -> fragment |> String.at(0) == ":" end)
+    |> Enum.uniq
+    |> Enum.sort
+    |> Enum.each(fn reserved ->
+      changeset = process_slug(reserved)
+      refute changeset.valid?, "#{reserved} should not be allowed as a slug"
```

Comment:
Specifying a fail message here. Otherwise, the test will fail with just `expected truthy, got nil`, with no information on which slug is failing.